modules/mirror-registry-release-notes.adoc

```diff
@@ -11,6 +11,15 @@ These release notes track the development of the _mirror registry for Red Hat Op
 
 For an overview of the _mirror registry for Red Hat OpenShift_, see xref:../../installing/disconnected_install/installing-mirroring-creating-registry.html#mirror-registry-flags_installing-mirroring-creating-registry[Creating a mirror registry with mirror registry for Red Hat OpenShift].
 
+[id="mirror-registry-for-openshift-1-2-3"]
+== Mirror registry for Red Hat OpenShift 1.2.3
+
+_Mirror registry for Red Hat OpenShift_ is now available with Red Hat Quay 3.7.4.
```

Comment:
Consider taking an RFE to add attributes for the most common "mirror registry for Red Hat OpenShift" permutations.